cypress/support/client.ts

```diff
@@ -57,6 +62,19 @@ declare global {
     }
 }
 
+Cypress.Commands.add("setUpKeyBackup", (): Chainable<void> => {
```

Comment:
This should probably live in settings.ts rather than client.ts given this doesn't deal with a client at all, or be ported to MatrixClient operations